Cargo.toml

```diff
@@ -44,6 +44,9 @@ sapp-wasm = { path ="./native/sapp-wasm", version = "=0.1.22" }
 [target.'cfg(not(any(target_os="linux", target_os="macos", target_os="android", target_os="ios", target_arch="wasm32", windows)))'.dependencies]
 sapp-dummy = { path ="./native/sapp-dummy", version = "=0.1.5" }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+webbrowser = "0.5.5"
```

Comment:
I am very concerned about this line. `miniquad` has no dependencies at all. Maybe this dependency should be under a feature? Or maybe remove this feature and make `link::open` working only on wasm? But, on other hand, this library has minimum dependencies too and works on Linux, Windows, MacOS, Android (!).

What do you think?